### ISSUE TITLE
[Feature Request] Update Reborn Blank Album icon

### ISSUE THREAD

**Issue by mariof1** [ampache/ampache#2181]
This is just a suggestion.
I think it is nicer to have just plain CD art rather than having a question mark embedded in it.

Something like this looks nice

<a href="/cliparts/D/O/1/I/0/8/cd-icon-th.png"><img src="http://www.clker.com/cliparts/D/O/1/I/0/8/cd-icon-md.png" alt='Cd Icon clip art'/></a>

**Comment by lachlan-00** [comment]
is it creative commons with an svg?
I probably won't be doing anything with the images but @kuzi-moto and others have been playing with themes a lot.

**Comment by AshotN** [comment]
I like the idea. But I'm pretty sure the API is what returns the default image. So it has to be updated server-side.

**Comment by kuzi-moto** [comment]
Yeah there are a couple images in reborn that don't seem to go well with the rest of the theme.

There is a section in the preferences to have a custom blank album image though. Under "admin" > "server config" > "interface" > "custom" > "Custom blank album default image"

**Comment by mariof1** [comment]
Yes, I am aware about this option although I could not find any documentation around size, picture format, paths. Then I though it would be nice to review a a few images with all of you and perhaps revamp them.

**Comment by AshotN** [comment]
The current file is a PNG file. I would stick with that as your export format. I'm not sure if you are planing on designing the image or just wanna pick one online. But if we do pick online we have to make sure there is no copyright and Ampache can actually use it.

**Comment by mariof1** [comment]
Here is one of the online links under public domain license.

https://publicdomainvectors.org/en/free-clipart/CD-symbol-with-reflection-vector-image/33891.html

The format is svg not sure if needs converting to png or can stay in current format.

**Comment by lachlan-00** [comment]
It would be good to put the theme effort into savage and replace it with svgs.

https://github.com/ampache/ampache/pull/2038

@kuzi-moto went through the effort of enabling svgs so let's put the effort into that

**Comment by mitchray** [comment]
I've remade the default image leaving out the question mark, which looks great alongside hi-res album art

![firefox_dJpoIHab20](https://user-images.githubusercontent.com/5735900/137617652-fe5fd841-3af6-49c2-abe6-d5e7706a1cf2.png)

Any objections?

**Comment by lachlan-00** [comment]
none, looks almost like a blank cd in a case.
takes me back to the mid 90s!
